Libplanet.Tests/Blockchain/BlockChainTest.ValidateNextBlock.cs

```diff
@@ -225,8 +282,62 @@ public void ValidateNextBlockInvalidStateRootHash()
                     _ => policyWithBlockAction.BlockAction,
                     stateStore,
                     new SingleActionLoader(typeof(DumbAction))));
+
             Assert.Throws<InvalidBlockStateRootHashException>(() =>
                 chain2.Append(block1, TestUtils.CreateBlockCommit(block1)));
+
+            chain1.Append(block1, TestUtils.CreateBlockCommit(block1));
+        }
+
+        [Fact]
+        public void ValidateNextBlockInvalidStateRootHashOnPostpone()
```

Comment:
New test for validate next block - when previous block is before-sloth, and appending block is after-sloth